Cargo.toml

```diff
@@ -347,6 +351,16 @@ description = "Illustrates bloom configuration using HDR and emissive materials"
 category = "3D Rendering"
 wasm = false
 
+[[example]]
+name = "debug_mesh"
+path = "examples/3d/debug_mesh.rs"
+
+[package.metadata.example.debug_mesh]
+name = "Debug mesh"
+description = "Debug meshes by displaying UVs, world normals, etc."
+category = "3D Rendering"
+wasm = false
```

Comment:
`wasm = false` is this an opt-out of having the example on Bevy's homepage?
I haven't tried the example on a wasm build, let me know what to put here.

Reply:
That's correct. I'd set this to `true`, it's visually impressive and good to verify.

Reply:
Set to true now.